client/rpc/block_converter.go

```diff
@@ -0,0 +1,115 @@
+package rpc
+
+import (
+	"encoding/json"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+	"github.com/tendermint/tendermint/types"
+)
+
+type ResultBlock struct {
```

Comment:
can we add godoc comments on each of the public structs and function?

Reply:
Added. Also unified how different types are written. Could you check please?

Reply:
Added